tardis/opacities/macro_atom/base.py

```diff
@@ -137,5 +361,7 @@ def _get_macro_atom_data(atomic_data):
             return atomic_data.macro_atom_data_all
 
 
-class NonMarkovChainTransitionProbabilities(TransitionProbabilities):
+class NonMarkovChainTransitionProbabilities(
```

Comment:
This comment is referring to line 327 in the file (can't comment on the unmodified code) -- The original `initialize_macro_atom_transition_type_filters` function in the class has no returning value, is it intended?

Reply:
never mind, the properties is updated using self.xxxx .